portfolio/src/components/Technologies-list.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Styled from 'styled-components';
+
+const ListContainer = Styled.ul`
+    display: flex;
+    padding: 0;
+    max-width: 100%;
+    flex-wrap: wrap;
+    list-style-type: none;
+    color: rgba(255,255,255,0.7);
+    li :not(:last-child) {
```

Comment:
:not(:last-child)

Reply:
I found this via google because i didnt know how to do this before yesterday, 

`:last-child` is a shortcut within CSS to apply styles to the last child in an array of elements.

`:not()` is the same thing as a `!` in javascript.

putting those two together `:not(:last-child)` means apply the style to all items EXCEPT the last one

notice the prefix of `li` so for this line of code it is saying apply this style to every `li` element except the last one. The style here is adding a border to the ride side of each element

this results in:

`C3 | Unity | Javascript`

Notice the `|` (the right border) doesn't show after `Javascript`